<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>room21</class>
 <widget class="QMainWindow" name="room21">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>5</y>
      <width>741</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Downloads/resource/desk.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Downloads/resource/puke/puke_total.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_25">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/resource/jetton/25.png</normaloff>../Downloads/resource/jetton/25.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_50">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>490</y>
      <width>90</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/resource/jetton/50.png</normaloff>../Downloads/resource/jetton/50.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_100">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>90</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/resource/jetton/100.png</normaloff>../Downloads/resource/jetton/100.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_player_bottom">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>460</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>player1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_player_top">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>player2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_wager">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>25</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_money">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_deal">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>deal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_double">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>510</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>double</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_hit">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>hit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_stand">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>500</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>stand</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>money:</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pb_25</zorder>
   <zorder>pb_50</zorder>
   <zorder>pb_100</zorder>
   <zorder>label_player_bottom</zorder>
   <zorder>label_player_top</zorder>
   <zorder>le_wager</zorder>
   <zorder>le_money</zorder>
   <zorder>pb_double</zorder>
   <zorder>pb_hit</zorder>
   <zorder>pb_stand</zorder>
   <zorder>label_5</zorder>
   <zorder>pb_deal</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
